File: Java/ArmstrongNumber.java
import java.util.*;
public class ArmstrongNumber
{
    public static void main(String argd[])
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a number");
        int num=sc.nextInt();
        int sum=0;
        int temp=num;
        int digit;

        while(temp>0)
        {
            digit=temp%10;
            sum=sum+(digit*digit*digit);
            temp=temp/10;
        }
        if(sum==num)
        {
            System.out.println(num+" is an Armstrong number");
        }
        else
        {
            System.out.println(num+" is not an Armstrong number");
        }
    }
}

File: C#/ArmstrongNumber.cs
using System;
class ArmstrongNumber
{
    static void Main(string[] args)
    {
        int num,sum=0,temp,rem;
        Console.WriteLine("Enter a Number:");
        num=Convert.ToInt32(Console.ReadLine());
        temp=num;

        while(num>0)
        {
            rem=num%10;
            sum=sum+rem*rem*rem;
            num=num/10;

        }
        if(sum==temp)
        {
            Console.WriteLine("It is an Armstrong Number");
        }
        else
        {
            Console.WriteLine("It is not an Armstrong Number");
        }
    }
}